docs/contributing/gatsby-docs-translation-guide.md

```diff
@@ -58,34 +61,57 @@ Maintainers of a translation are expected to review translations for quality, fr
 
 Translation contributions must be opened as pull requests to provide time for review and comments. If actionable feedback is given on a PR review, the author must acknowledge the review and make changes with their discretion. Ignoring review feedback completely is not allowed (see response templates below for help with this).
 
-### Maintainer responsibilities
+## Maintainer responsibilities
 
 As repo maintainers and members of the Gatsby community, your responsibilities are as follows:
 
-- Keep issues up to date as people volunteer to translate pages.
+- Keep issues up-to-date as people volunteer to translate pages.
 - Review pull requests made by contributors promptly.
-- Review pull requests generated by gatsbybot in order to make sure translations remain up to date with the source repo. (details to come)
+- Review pull requests generated by gatsbybot in order to make sure translations remain up-to-date with the source repo. (details to come)
 - Act as point of contact for your language and answer questions from both contributors to your language and the core Gatsby team.
 - Set up a process in order to get your translation published. (details to come)
 
 As a maintainer, you are welcome to add a contributing doc written in your language to assist with the process. You can find an example in the [gatsby-es repo](https://github.com/gatsbyjs/gatsby-es/blob/master/CONTRIBUTING.MD). Translating [this page](https://github.com/gatsbyjs/gatsby/blob/master/docs/contributing/gatsby-docs-translation-guide.md) and copying it into a `contributing.md` file would be an option as well.
 
-Additional tips:
+### Additional tips
+
+#### Set up a style guide and glossary
+
+Your language repo comes with a template [style guide](https://github.com/gatsbyjs/gatsby-i18n-source/blob/master/style-guide.md) that you can use to put in style rules specific to your language. Refer to the [translation style guide](#translation-style-guide) section for more information.
+
+#### Collaborate on difficult pages
+
+Some pages are very long and difficult to translate for a single contributor. Feel free to split these pages up into sections and work on them together!
+
+In addition, sometimes a contributor cannot continue a PR for whatever reason. Feel free to ask them if they need help and to bring someone else in to finish the work for them!
```

Comment:
Changing up the wording a bit:
```suggestion
In addition, sometimes a contributor cannot continue a PR for whatever reason. It may help to ask them if they need assistance and to bring someone else on to complete the work in a timely fashion.
```